lib/matplotlib/tests/test_axes.py

```diff
@@ -1464,6 +1464,14 @@ def test_hist_step_filled():
     assert all([p.get_facecolor() == p.get_edgecolor() for p in patches])
 
 
+@image_comparison(baseline_images=['hist_density'], extensions=['png'])
+def test_hist_density():
+    np.random.seed(19680801)
+    data = np.random.standard_normal(2000)
+    fig, ax = plt.subplots()
+    ax.hist(data, density=True)
```

Comment:
Does this actually need an image test, or can we just test that `np.histogram` properly applies the density kwarg?

Reply:
I don't think there was ever any image test that used the 'density' or 'norm' `kwargs`, so I thought it would be a good idea to add one.

Reply:
I think we are trying to keep image tests to things where the actual image appearance might change if code changes.  So sure if there is no hist image test there should be one.  If such a test does exist then this just tests the value of the bars, and I think you can do that w/o an image comparison.   But I’m not passionate about it or anything.

Reply:
Beating a dead horse, but:

```python

np.random.seed(19680801)
data = np.random.standard_normal(2000)
fig, ax = plt.subplots()
n, bins, patches = ax.hist(data, density=True)
assert np.isclose(np.sum(n * np.diff(bins)), 1.0)
```